test/integration/searchRanking.js

```diff
@@ -0,0 +1,207 @@
+const cheerio = require('cheerio');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+const resultsRoute = `${constants.SITE_ROOT}/results/`;
+
+function assertSearchResponse(search, assertions) {
```

Comment:
This function name feels like it could be improved for future reference. Something like `makeSearchRequestAndCheckExpectations` perhaps?